### DISCUSSION TITLE
How to init data to database ?

### DISCUSSION THREAD

**Discussion by Paul75** [TypedProject/ts-express-decorators#1803] [Q&A] [answered]
Hello,

When I can initialize the database ?

Thanks

**Comment by Romakita** [comment]
$onInit or $onReady

**Comment by Paul75** [comment, marked as answer]
Hello,

In the documentation it is explain ? I use TypeOrm.

Thanks

**Reply by Romakita** [reply]
No because, it's related to Typeorm, not directly to Ts.ED.

**Reply by Paul75** [reply]
ok
